internal/monitors/cadvisor/metadata.yaml

```diff
@@ -0,0 +1,437 @@
+  - description: The number of microseconds that the [CFS scheduler](https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt)
+      uses as a window when limiting container processes
+    included: true
+    name: container_spec_cpu_period
+    type: gauge
+  - description: In CPU quota for the CFS process scheduler. In K8s this is equal
+      to the containers's CPU limit as a fraction of 1 core and multiplied by the
+      `container_spec_cpu_period`.  So if the CPU limit is `500m` (500 millicores)
+      for a container and the `container_spec_cpu_period` is set to 100,000, this
+      value will be 50,000.
+    included: true
+    name: container_spec_cpu_quota
+    type: gauge
+  - description: CPU share of the container
+    included: false
+    name: container_spec_cpu_shares
+    type: gauge
+  - description: Memory limit for the container.
+    included: true
+    name: container_spec_memory_limit_bytes
+    type: gauge
+  - description: Memory swap limit for the container.
+    included: false
+    name: container_spec_memory_swap_limit_bytes
+    type: gauge
+  - description: Start time of the container since unix epoch in seconds.
+    included: false
+    name: container_start_time_seconds
+    type: gauge
+  - description: Number of tasks in given state
+    included: false
+    name: container_tasks_state
+    type: gauge
+  - description: Number of CPU cores on the node.
+    included: true
+    name: machine_cpu_cores
+    type: gauge
+  - description: Node's CPU frequency.
+    included: false
+    name: machine_cpu_frequency_khz
+    type: gauge
+  - description: Amount of memory installed on the node.
+    included: true
+    name: machine_memory_bytes
+    type: gauge
+  - description: Cumulative count of bytes received
+    included: true
+    name: pod_network_receive_bytes_total
+    type: counter
+  - description: Cumulative count of errors encountered while receiving
+    included: true
+    name: pod_network_receive_errors_total
+    type: counter
+  - description: Cumulative count of packets dropped while receiving
+    included: false
+    name: pod_network_receive_packets_dropped_total
+    type: counter
+  - description: Cumulative count of packets received
+    included: false
+    name: pod_network_receive_packets_total
+    type: counter
+  - description: Cumulative count of bytes transmitted
+    included: true
+    name: pod_network_transmit_bytes_total
+    type: counter
+  - description: Cumulative count of errors encountered while transmitting
+    included: true
+    name: pod_network_transmit_errors_total
+    type: counter
+  - description: Cumulative count of packets dropped while transmitting
+    included: false
+    name: pod_network_transmit_packets_dropped_total
+    type: counter
+  - description: Cumulative count of packets transmitted
+    included: false
+    name: pod_network_transmit_packets_total
+    type: counter
+  monitorType: cadvisor
+  properties: null
+- dimensions:
```

Comment:
Since this YAML isn't auto-generated, use YAML anchor/refs to dedup this as much as possible.  I think the two monitors have exactly the same output.